### ISSUE TITLE
undefined symbol: RBASIC_CLASS

### ISSUE THREAD

**Issue by brianphillips** [tmm1/rbtrace#36]
I get an error in my unicorn log (presumably) when rbtrace hits a certain type of method call in the process it's tracing.  The error is:
`<snip>/vendor/bundle/ruby/2.0.0/gems/rbtrace-0.4.6/lib/rbtrace.so: undefined symbol: RBASIC_CLASS`

The version of ruby I'm running:
```
$ ruby -v
ruby 2.0.0p353 (2013-11-22 revision 43784) [x86_64-linux]
```

Any suggestions on if this is something I can fix myself?

**Comment by tmm1** [comment]
This is fixed in v0.4.7

**Comment by brianphillips** [comment]
wow, was not expecting a resolution so quickly.  Thanks! :+1:
